dts/arm/nordic/nrf54l15_cpuapp.dtsi

```diff
@@ -44,15 +49,23 @@ nvic: &cpuapp_nvic {};
 		interrupts = <76 NRF_DEFAULT_IRQ_PRIORITY>;
 		#mbox-cells = <1>;
 		nordic,events = <1>;
+#ifdef NRF54L15_ENGA
 		nordic,events-mask = <0x00008000>;
+#else
+		nordic,events-mask = <0x00100000>;
+#endif
```

Comment:
does this work? where is this define injected?

why not simply have a common file for enga/non-enga, and a specific file with the necessary overrides for each?

Reply:
done by override